conf/local.conf.sample

```diff
@@ -96,21 +96,6 @@ BB_NUMBER_THREADS ?= "${@int(oe.utils.cpu_count() * 1.5)}"
 # How many jobs 'make' should run in parallel when compiling
 PARALLEL_MAKE ?= "-j ${@oe.utils.cpu_count() * 2}"
 
-# Uncomment to exclude GPLv3 software from the build
-#INCOMPATIBLE_LICENSE = "GPL-3.0* LGPL-3.0*"
```

Comment:
@chombourger it was decided that the option will remain there in local.conf to build with non GPLV3

Reply:
@Noor-Ahsan Ah, if that's the case then this PR should be dropped, as it won't be usable without the rest.